pages/common/nvm.fish.md

```diff
@@ -0,0 +1,25 @@
+# nvm.fish
```

Comment:
If I understand correctly, the binary is `nvm` so I think there is a mismatch here.
Moreover, the CLI interface seems similar to https://github.com/nvm-sh/nvm so I am not sure this whole page is needed. 💭

Reply:
There are several differences between the original `nvm` with this version. In short, currently the commands between both versions aren't compatible with each other:

1. No `nvm run` or `nvm exec` (though I can just create a PR for that 😀)
2. The default Node.js version is set through `set` instead of a subcommand (again, even though I can just PR about it 😅)

Maybe I should close this PR by sending a couple to https://github.com/jorgebucaran/nvm.fish...

Reply:
Thanks for the detailed information. :+1: 
I understand that command are different. :ok_hand: 
But I don't understand what to do with the mismatch between the binary name (`nvm`) and the page name (we would need to run `tldr nvm.fish` to find it, if I understand correctly). :thinking:

Reply:
There's a bunch of related discussions related to this:

1. #7480 - The one I've referenced here
2. #7503 - Plans to separate commands by shells, if so I can move this into something like `common/fish/nvm`
3. #6621 - PowerShell, related to #7503

Reply:
The title of a page needs to exactly match the command name being executed. This is not the case for the filename.

 - **Filename:** `nvm.fish.md`
 - **Page title:** `nvm`
 - **Command being executed:** `nvm`

Ideally the page title and the filename will match, but this is unfortunately not always possible.

Reply:
For completeness, the command in question where it's also not possible is `rename`.